sdk/dotnet/Pulumi.Gcp.csproj

```diff
@@ -45,7 +45,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi" Version="3.*" />
+    <PackageReference Include="Pulumi" Version="[3.66.1.0,4)" />
```

Comment:
I don't think this is correct in .NET as using a specific range will make nuget resolve to the **lowest** possible version.